docs/ibm-legacy-rules.md

```diff
@@ -121,27 +119,6 @@ The supported rules are described below:
 #### Statuses
 
 Each rule can be assigned a status. The supported statuses are `error`, `warning`, `info`, `hint` and `off`.
-Some rules can be configured further with configuration options. The format of this configuration is to provide an array, rather than just a string. e.g.
```

Comment:
I removed this configuration section because the paths_case_convention rule was our last remaining legacy rule that supported any sort of configuration other than just the status.